src/components/input/SimpleInputText.vue

```diff
@@ -53,10 +54,12 @@ export default {
       return this.label.length > 0 ? {['outlined']: true} : {['solo']: true}
     },
     getRules() {
-      return validationRules[this.rule]
+      const rulesToApply = []
+      this.rules.forEach(rule => rulesToApply.push(validationRules[rule]))
+      return rulesToApply
```

Comment:
ici, tu peux refacto en utilisant un .map comme ça, ça te fait une ligne à la fin: 

`getRules() {
  return this.rules.map((rule) => validationRules[rule])
}`